src/assets/scss/mixin.scss

```diff
@@ -0,0 +1,58 @@
+@import './_variables.scss';
+@mixin d_flex {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+@mixin d_flex_column {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+}
+@mixin max_width {
+  width: auto;
+  max-width: $w_100;
+}
+@mixin position_abs_left {
+  position: $position_abs;
+  left: 0;
+  top: 0;
+}
+@mixin transition {
+  transition: all 0.2s ease;
+  -webkit-transition: all 0.2s ease;
+  -moz-transition: all 0.2s ease;
+}
+@mixin position_abs_right {
+  position: $position_abs;
+  right: 0;
+  top: 0;
+}
+@mixin _before_position_abs{
+  content:"";
+  @include position_abs_left;
+}
+@mixin _after_position_abs{
+  content:"";
+  @include position_abs_left;
+}
+@mixin flex_ul {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0;
+  margin: 0;
+}
+@mixin user_img {
+  border-radius: 100%;
+  object-fit: cover;
+}
+@mixin list_style_none {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+@mixin font_face_design {
+  font-style: normal;
+  font-display: swap;

```

Comment:
please write down its benefit so we can include it in our stylesheet.

Reply:
swap gives the font face a zero-second block period and an infinite swap period. This means the browser draws text immediately with a fallback if the font face isn’t loaded, but swaps the font face in as soon as it loads.